tests/serial/map-if-necessary/three-solvers/mixed-substeps/ParallelExplicit.cpp

```diff
@@ -0,0 +1,31 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(ThreeSolvers)
+BOOST_AUTO_TEST_SUITE(MixedSubsteps)
+
+BOOST_AUTO_TEST_CASE(ParallelExplicit)
+{
+  PRECICE_TEST("A"_on(1_rank), "B"_on(1_rank), "C"_on(1_rank));
+
+  std::vector<int> readMappings{2, 2, 2};
```

Comment:
```suggestion
  std::vector<int> readMappings{3, 3, 3};
```
1 from A, 2 from B?

Reply:
Parallel explicit doesn't send substeps as they would never be reachable.

So 2, 2, 2 is correct.